### ISSUE TITLE
Ekaterina Mamina: UX/UI, 2 weeks

### ISSUE THREAD

**Issue by katsmamina** [HackYourFutureBelgium/class-15#168]
# Study plan

## User Experience and User Interfaces

- [ ] Empathy for Developers

- [ ] UX Research, User Interviews & Personas

- [ ] Design Thinking

- [ ] Design Thinking for Developers

## General

- [ ] User experience (UX)

- [ ] Gestalt

- [ ] Colour

- [ ] Iconography

- [ ] Typography and Fonts

- [ ] Style Guides

- [ ] Accessibility

- [ ] Imagery and Photography

- [ ] Illustration

- [ ] Design 

- [ ] inspiration

- [ ] Free Courses

- [ ] Figma

- [ ] UI Kits

- [ ] More Softwares

- [ ] Books

## Open Source Software

- [ ] What is Open Source Software?

- [ ] Open Source Licenses

- [ ] Codes of Conduct

- [ ] Contributor Guidelines

- [ ] How to Contribute

> [module repository](_)

**Comment by katsmamina** [comment]
Thank you @WalterAlvar ! Added

**Comment by katsmamina** [comment]
Week 2

- [x] make project in figma
- [ ] [project issue](your-link-here) (if there is a project)


### Check-In

#### I Need Help With:
- so far with nothing

#### What went well?
- Creating CSS and HTML for my website
- playing with Figma

#### What went less well?
- after the Sunday class I totally reimagined design of my website, so the work of the previous week was mostly useless.
- the given time is not enough for learning Figma and CSS deep enough. I feel that I'm spending too much time and still don't progress much in learning new things 😭 

#### Lessons Learned
- how to use Figma
- CSS Grid and Media queries (still in process!) 

### Sunday Prep Work
- contribute to other students' projects
- make my website responsive

**Comment by colevandersWands** [comment]
> the given time is not enough for learning Figma and CSS deep enough. 

That's true, to give either of those topics enough time you'd need a full course.  You'll get many chances to practice them in later modules and the final project

>  feel that I'm spending too much time and still don't progress much in learning new things 😭

If you redesigned your website, then I'd guess you learned something about design!
